src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="content">
+    <section class="content__calendar calendar calendar--start-day-mon calendar--month-length-31">
```

Comment:
You don't need to make content as a BEM-block.It used only for positioning the calendar on the page.